cryptoorgchain/chain.json

```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "../chain.schema.json",
-  "chain_name": "cryptoorgchain",
+  "chain_name": "cronosposchain",
```

Comment:
Can we please leave the chain_name alone? but everything else is fine to change.

Reply:
Thanks @JeremyParish69! Centernally, and keeping it as `cryptoorgchain`